src/pages/overview/pdf-services-api/gettingstarted.md

```diff
@@ -322,7 +322,7 @@ Jumpstart your development by bookmarking or downloading the following key resou
 
 The samples project requires the following:
 
--   NET: version 6.0 or above
+-   NET: version 8.0 or above
 -   A build Tool: Either Visual Studio or .NET Core CLI.
 
 #### Authentication
```

Comment:
line 332 -> .NET Samples instead of `adobe-dc-pdf-services-sdk-java-samples`

Reply:
Fixed